crates/rpc-types-engine/src/envelope.rs

```diff
@@ -24,32 +74,7 @@ pub struct OpNetworkPayloadEnvelope {
 
 impl OpNetworkPayloadEnvelope {
     /// Decode a payload envelope from a snappy-compressed byte array.
-    /// The payload version decoded is `ExecutionPayloadV1` from SSZ bytes.
-    #[cfg(feature = "std")]
-    pub fn decode_v1(data: &[u8]) -> Result<Self, PayloadEnvelopeError> {
```

Comment:
Dumb Q - but why was this removed?

Reply:
because op doesn't support v1 https://specs.optimism.io/protocol/exec-engine.html can be out of scope to fix this semantic in this pr